templates/events/source-events.html

```diff
@@ -23,12 +23,12 @@ <h1>{% block title %}Event Sources{% endblock %}</h1>
 {% block content %}
 
 <nav aria-label="breadcrumb">
-  <ol class="breadcrumb">
+  <ol class="breadcrumb calendar-event-breadcrumb">
     <li class="breadcrumb-item active" aria-current="page">{{title}}</li>
   </ol>
 </nav>
 
-<h1 class="text-center">{{title}}</h1>
+<h1 class="text-center calendar-event-title">{{title}}</h1>
```

Comment:
The calendar-event-title is not reflected in "Add New Calendars" title
![Screen Shot 2022-11-03 at 10 20 01 AM](https://user-images.githubusercontent.com/22112888/199762057-ee72b3d5-ab2b-479f-8535-11cd18ec12f9.png)

Reply:
I think this pull request only updates the style of the event listing page, and the setting page is not included since we don't have a design for it on Zeplin.

Reply:
sounds good.